fitbenchmarking/fitting/mantid/externals.py

```diff
@@ -61,9 +61,15 @@ def set_ties(function_object, ties):
 
     @returns :: mantid function object with ties
     """
+
     for idx, ties_per_func in enumerate(ties):
         for tie in ties_per_func:
-            exec "function_object.tie({'f" + str(idx) + "." + tie + "})"
+            exec("param_dict = {'f" + str(idx) + "." + tie + "}")
```

Comment:
param_dict appears to be declared but not used. Please delete commented out lines of code that are no longer needed. Please add a code comment to explain this more complex code